src/gt4py/__init__.py

```diff
@@ -20,6 +20,10 @@
 __license__ = "GPLv3+"
 
 
+from pkg_resources import DistributionNotFound, get_distribution
+
+
+# isort: off
```

Comment:
Out of interest: what would isort do here if it was enabled?

Reply:
It would sort this imports alphabetically, which would mess up with the internal dependencies between modules